spec/v0-1.md

```diff
@@ -66,6 +66,10 @@ The expression `(quote <datum>)` or `'<datum>` evaluates to `<datum>`.
 
 The expression `(atom <e>)` evaluates to `t` if `<e>` evaluates to an atom (non-list) and to `nil` if `<e>` evaluates to a non-atom (list).
 
+### `procedure`
```

Comment:
Can we just use 'function' instead of 'procedure'? The type in `lurk-rs` is `Fun`, and Common Lisp calls things like this 'functions'.

Reply:
Yes, that works.

Reply:
Done!

Reply:
Sorry… looking a little more closely, `functionp` implies a predicate. In CL it returns T or NIL. If this needs to return the function itself, it's a bit of a misnomer.

Reply:
This is exactly a predicate!

Reply:
Technically it (the CL function) doesn't need to return T, but it's not required to return the function itself.

Reply:
> This is exactly a predicate!

I see, yes, then the name is perfect. Something misled me down that tangent when I read before — but I couldn't tell you what. In any case, all is good.